### PR TITLE
updated CB tutorial

### PR DESCRIPTION
**Pull request by mikedropmedia**
Mike's first draft edits

### CODE REVIEW

**Comment by mikedropmedia** [comment]
Cheng, can we look at this doc together quickly to check for white space or errors, or whatever was holding us up Friday? I used Atom to push a new version (I will work in smaller chunks of documentation and text moving forward so it's not an entire article commit...)

Mike

File: _guides/contextual_bandits.md [hunk 1]
```diff
@@ -8,108 +8,114 @@ show_on_learn: false
 layout: tutorial
 body_class: tutorial
 resource_icon: /svg/resources/guide.svg
----
```
Comment (cheng-tan) [inline _guides/contextual_bandits.md]: Need to keep the dash, this is required for the framework we are using.

**Comment by mikedropmedia** [comment]
Added Getting started with VW and command line Tutorial links (new H2 section and updated build and package Note).

File: _guides/contextual_bandits.md (hunk 1 above)
Comment (mikedropmedia) [reply _guides/contextual_bandits.md]: I don't see line 11 on my version. It's a blank line. So I add one dash is all?

**Review by mikedropmedia — COMMENTED** [review]
I don't understand the dash request.

File: _guides/contextual_bandits.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ This guide describes how to run Contextual Bandit (CB) algorithms in Vowpal Wabb
 
 To install VW—and for more information on building VW from source or using a package manager—see [Getting started](https://github.com/VowpalWabbit/vowpal_wabbit/wiki/) guide. Use this [Tutorial](https://github.com/VowpalWabbit/vowpal_wabbit/wiki/Tutorial) for a step-by-step guide to VW command line mode if you are ready to jump in.
 
->**Note:** The CB tutorial uses [Vowpal Wabbit Python package](https://github.com/VowpalWabbit/vowpal_wabbit/tree/master/python). VW is supported in C++ and C#, and additional binary packages are available for select platforms. See Getting started module on the homepage for more information.
+>**Note:** The CB tutorial uses [Vowpal Wabbit Python package](https://github.com/VowpalWabbit/vowpal_wabbit/tree/master/python). VW is supported in C++ and C#, and additional binary packages are available for select platforms. See [Getting started](https://cheng-tan.github.io/vowpalwabbit.github.io/) module on the homepage for more information.
```
Comment (cheng-tan) [inline _guides/contextual_bandits.md:21]: this needs to be a relative URL, I'll send it to you tomorrow.

**Pull request merged**